nixosModules/hermes/chain-options.nix

```diff
@@ -0,0 +1,255 @@
+{lib}:
```

Comment:
Can you add a comment here explaining what these configuration options represent, and maybe link some documentation about them?

Reply:
yes, will attach hermes doc if it exists (hopefully). 

@JonathanLorimer what way to add doc comments into nix?

so people do it 10 different ways.